solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -74,6 +75,8 @@
 
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
+  private static final Pattern COMMA_SEPARATED_PATTERN = Pattern.compile(",\\s*");
```

Comment:
Should this accept whitespace on either side of the comma? `\\s*,\\s*`

Reply:
Yes